### PR TITLE
Add algorithm types to threat detection disabled alerts description

### PR DESCRIPTION
**Pull request by t-arrinb**
Added Data_Exfiltration and Unsafe_Action as allowed values to disabled alert.

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Review by sergey-shandar — APPROVED** [review]

**Comment by sergey-shandar** [comment]
@dnayantara @jaredmoo @nathannfan please, review.

File: specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/serverSecurityAlertPolicies.json [hunk 1]
```diff
@@ -159,7 +159,7 @@
           }
         },
         "disabledAlerts": {
-          "description": "Specifies an array of alerts that are disabled. Allowed values are: Sql_Injection, Sql_Injection_Vulnerability, Access_Anomaly",
+          "description": "Specifies an array of alerts that are disabled. Allowed values are: Sql_Injection, Sql_Injection_Vulnerability, Access_Anomaly, Data_Exfiltration, Unsafe_Action",
```
Comment (jaredmoo) [inline specification/sql/resource-manager/Microsoft.Sql/preview/2017-03-01-preview/serverSecurityAlertPolicies.json]: These possible values should be defined using "enum" and "x-ms-enum" properties.

**Review by jaredmoo — CHANGES_REQUESTED** [review]
Description is good but we should also have enum defined. (Since there are more possible values in future, use modelAsString: true).

**Comment by sergey-shandar** [comment]
@t-arrinb could you address comments?

**Comment by t-arrinb** [comment]
Altering to enum may have larger implications than just here (requiring RP change as well). I'm discussing with my manager today on how to proceed.

**Comment by t-arrinb** [comment]
Currently we'll stay with description as is, opened task 294228 to move to enum.

**Comment by t-arrinb** [comment]
@sergey-shandar What's the next step?

**Pull request merged**